src/mol-math/geometry/lookup3d/common.ts

```diff
@@ -41,8 +41,10 @@ export namespace Result {
 export interface Lookup3D<T = number> {
     // The result is mutated with each call to find.
     find(x: number, y: number, z: number, radius: number, result?: Result<T>): Result<T>,
+    nearest(x: number, y: number, z: number): { index: number, squaredDistance: number } | undefined,
+    distanceTo(x: number, y: number, z: number): number,
```

Comment:
Instead of this, add function ``distanceToVec(sphere: Sphere3D, v: Vec3)`` to ``namespace Sphere3D``.